src/lib/homeobject_impl.hpp

```diff
@@ -63,6 +63,7 @@ struct PG {
 
     PGInfo pg_info_;
     uint64_t shard_sequence_num_{0};
+    std::atomic_uint64_t blob_sequence_num_{0};
```

Comment:
Can we do:
```c++
std::atomic<blob_id_t> blob_sequence_num_{0};
```
So it's clear what it's value is representing?

Reply:
We can also remove the `shard_seq_num_` from the `ShardIndex` in memory implementation if we have a base counter now right?